.buildkite/scripts/steps/scalability/benchmarking.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+#.buildkite/scripts/bootstrap.sh
+echo "--- yarn kbn reset && yarn kbn bootstrap"
+yarn kbn reset && yarn kbn bootstrap
+
+GCS_ARTIFACTS_DIR="gcs_artefacts"
```

Comment:
```suggestion
GCS_ARTIFACTS_DIR="gcs_artifacts"
```
Guess artifact is commonly used one in kibana repo even though the other one is also correct